core/common/src/geometry/GeometryStream.ts

```diff
@@ -435,7 +435,7 @@ export class GeometryStreamBuilder {
     return undefined !== hdr && GeometryStreamFlags.None !== (hdr.flags & GeometryStreamFlags.ViewIndependent);
   }
   public set isViewIndependent(viewIndependent: boolean) {
-    if (viewIndependent === this.isViewIndependent)
+    if (viewIndependent === this.isViewIndependent && this.getHeader() !== undefined)
```

Comment:
obtainHeader creates the header and inserts it at the head. The header is never present by default...with this change viewIndependent will always be ignored...

Reply:
issues like this is why i'm very hesitant to encourage this whole change. Why not just rework the EC part of the native code? @khanaffan?

Reply:
The goal of this change was to ensure compatibility with current getElement output. I think the default behavior could remain unchanged if we added a separate method that would explicitly request appending header information

Reply:
The current output always includes the header. Now you will never include the header. For compatibility obtainHeader will need to be called followed by a call to set isViewIndependent (original version!) if needed.

```
void GeometryCollection::ToJson(BeJsValue output, BeJsConst opts) const
    {
    output.SetNull();
    GeometryStreamIO::Collection collection(m_data, m_dataSize);
    GeometryStreamIO::Reader reader(m_state.m_dgnDb);
    DgnSubCategoryId lastSubCategory = m_state.m_geomParams.GetSubCategoryId();
    bool wantBRepData = opts["wantBRepData"].asBool();

    for (auto const& egOp : collection)
        {
        switch (egOp.m_opCode)
            {
            case GeometryStreamIO::OpCode::Header:
                {
                auto header = output.appendValue()["header"];
                header["flags"] = static_cast<uint32_t>(reader.GetHeader(egOp)->m_flags);
                break;
                }
...
```